docs/README.md

```diff
@@ -87,9 +87,10 @@ This only runs the diracx server, not any dependency like external IdP.
 
 ## Add a DB
 
-Database classes live in `src/diracx/db/<dbname>`. Have a look at the `src/diracx/db/dummy/` to see how to implement your own DB
+Database classes live in `src/diracx/db/sql/<dbname>`. Have a look at the `src/diracx/db/sql/dummy/` to see how to implement your own DB.
```

Comment:
You also need to include opensearch DBs (e.g. https://github.com/DIRACGrid/diracx/blob/main/diracx-db/src/diracx/db/os/job_parameters.py)

What about: 
```suggestion
Database classes live in `src/diracx/db/<dbtype>/<dbname>`. Have a look at the `src/diracx/db/sql/dummy/` to see how to implement your own DB.
```